drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -240,6 +240,19 @@ end
 
 local function device_init(driver, device)
   device:subscribe()
+  local windMode_support = false
+  -- To check if the device support WindSupport WindSupportMask
+  local heat_eps = device:get_endpoints(clusters.FanControl.ID, {feature_bitmap = clusters.FanControl.types.WindSupportMask.SLEEP_WIND})
+  if #heat_eps ~= 0 then
+    windMode_support = true
```

Comment:
considering the following change

 local heat_eps = device:get_endpoints(clusters.FanControl.ID, {feature_bitmap = clusters.FanControl.types.WindSupportMask.SLEEP_WIND})
local fan_without_windmode profile  = fan-without-windmode"
  if #heat_eps == 0 then
    device:try_update_metadata({profile = fan_without_windmode profile})